dts/arm/st/f4/stm32f410.dtsi

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <zephyr/dt-bindings/clock/stm32f4_clock.h>
```

Comment:
See that some other stm32f4x devices have a DCKCFGR2 : stm32f410, f412, f413, f423